src/test/javascript/spec/component/shared/code-button.component.spec.ts

```
@@ -451,4 +477,135 @@ describe('CodeButtonComponent', () => {
+        const inputs = form.getElementsByTagName('input');
+        const data: { [key: string]: string } = {
+            appDef: 'theia-image',
+            gitUri: 'https://admin@repo.uri',
+            gitToken: 'token',
+        };
+
+        for (const key in data) {
+            if (Object.hasOwn(data, key)) {
+                const input = Array.from(inputs).find((input) => input.name === key);
+                expect(input).toBeDefined();
+                expect(input!.value).toBe(data[key]);
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)